.clang-format

```diff
@@ -1,4 +1,226 @@
-# Defines the Chromium style for automatic reformatting.
-# http://clang.llvm.org/docs/ClangFormatStyleOptions.html
-BasedOnStyle: Chromium
-Standard: Cpp17
+---
+Language:        Cpp
```

Comment:
You can generate this file using clang-format -style=Chromium -dump-config > .clang-format